helga_jeopardy.py

```diff
@@ -298,13 +556,109 @@ def jeopardy(client, channel, nick, message, cmd, args,
         reset_channel(channel, mongo_db)
         return 'done'
 
+
     # if we have an active question, and args, evaluate the answer
 
     question = mongo_db.find_one({
         'channel': channel,
         'active': True,
     })
 
+    new_game = mongo_db.find_one({
+        'channel': channel,
+        'game_active': True,
+        'game_started': False,
+    })
+
+    current_game = mongo_db.find_one({
+        'channel': channel,
+        'game_active': True,
+        'game_started': True,
+    })   
+
+    if len(args) > 0 and args[0] == 'game':
+        if args[1] == 'new':
+            if new_game or current_game:
+                client.msg(channel, "Game already created or in progress, the host can end the current game with: !j game end")
+                return
+            else:
+                client.msg(channel, "Fetching questions and categories from the API. This will take ~10s")
```

Comment:
Did you say you wanted to update this to indicate that it might take closer to 15 seconds?